app/css/style.css

```diff
@@ -1,2 +1,92 @@
+* {
+  font-family: 'Raleway', sans-serif; }
 
-/*# sourceMappingURL=data:application/json;charset=utf8;base64,eyJ2ZXJzaW9uIjozLCJuYW1lcyI6W10sIm1hcHBpbmdzIjoiIiwic291cmNlcyI6WyJzdHlsZS5zY3NzIl0sInNvdXJjZXNDb250ZW50IjpbIiJdLCJmaWxlIjoic3R5bGUuY3NzIn0= */
+.nav {
+  background-color: #ffffff;
+  z-index: 10;
+  height: 50px;
+  width: 100%;
+  position: fixed; }
+  .nav .nav-link {
+    color: black;
+    padding-top: 1.8rem;
+    font-size: 1.2rem; }
+  .nav #title {
+    font-size: 2rem;
+    padding-top: 1rem;
+    background: -webkit-linear-gradient(right, #d7d2cc, #304352);
+    -webkit-background-clip: text;
+    -webkit-text-fill-color: transparent; }
+
+.nav-transform {
+  height: 85px;
+  transform: rotate(1.5deg);
+  position: fixed;
+  width: 110%;
+  top: 0;
+  left: 0;
+  background-color: #ffffff;
+  z-index: 5; }
+
+aside {
+  height: 100vh;
+  background-image: -webkit-linear-gradient(#d7d2cc, #304352);
+  overflow: auto; }
+
+body {
+  overflow: hidden; }
+
+#accordionExample {
+  text-align: left;
+  background-image: -webkit-linear-gradient(#ffffff, #d7d2cc);
+  border-radius: 5px; }
+  #accordionExample .card-body {
+    padding: 5px 0 5px 30px; }
+  #accordionExample .card {
+    background: transparent;
+    color: #3E5151; }
+    #accordionExample .card .collapse {
+      text-align: left; }
+  #accordionExample .btn-link {
+    font-weight: bold;
+    color: #3E5151; }
+  #accordionExample .btn-link:hover {
+    text-decoration: none; }
+
+#carouselExampleControls {
+  padding: 0; }
+
+.carousel-inner img {
+  width: 100vw;
+  height: 100vh;
+  object-fit: cover;
+  overflow: hidden; }
+
+#getRecipeBtn {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  margin-bottom: 25px;
+  padding: 5px 0;
+  background-color: #3ae8a5;
+  color: #304352;
+  z-index: 100; }
```

Comment:
Can see 5, 10, and 100 on the z-index. 

should this follow a more regular pattern, ideally 100, 200, 300 respectively?

Reply:
renamed z-index instances